### ISSUE TITLE
Agregar alternativa de reportar bugs a usuarios sin cuenta en github

### ISSUE THREAD

**Issue by christianrd** [Emplea-do/emplea_do#47]
Agregar la opción para usuarios que no saben o no usan github para reportar bugs de emplea.do
Idea formulada por @eatskolnikov :

- [x] Formulario en el sitio web
- [x] Implementar Captcha
- [ ] Al mandarlo creamos un issue en github
- [ ] Le ponemos un tag de "User submitted"
- [ ] Notificamos en un canal de slack

**Comment by eatskolnikov** [comment]
@christianrd ¿Qué sugerencias tienes?

**Comment by christianrd** [comment]
@eatskolnikov creo que se puede usar los famosos formularios de google, microsoft, entre otros... o crear una vista que envie eso como mensaje a un correo destinado para eso de emplea.do.

**Comment by eatskolnikov** [comment]
🤔 A ver que se le ocurre a la comunidad, comparte este issue en el slack y dale mention @channel

**Comment by HommyDeJ** [comment]
Opino que se podría crear un contact form que contenga nombre de la persona que está reportando, obviamente el bug. Esos mensajes llegarían a una cuenta de correo manejada por uno de los admins del site.

**Comment by christianrd** [comment]
Deacuerdo con @HommyDeJ pero para hacerlo mas interesante que llegue como notificacion al canal de Slack del proyecto.

**Comment by eatskolnikov** [comment]
Estoy de acuerdo con crear una notificación en slack, no tanto con mandar un correo. ¿quién va a atender ese buzón? 🤔

**Comment by eatskolnikov** [comment]
Chequeen esta idea:
- Formulario en el sitio web
- Al mandarlo creamos un issue en github
- Le ponemos un tag de "User submitted"
- Notificamos en un canal de slack

**Comment by HommyDeJ** [comment]
@eatskolnikov  Me parece muy buena la idea, puedo tomar este issuse?

**Comment by eatskolnikov** [comment]
@HommyDeJ esto depende del issue #46

**Comment by HommyDeJ** [comment]
@eatskolnikov Si, entonces debo esperar a que se cierre ese ticket?

**Comment by christianrd** [comment]
Volviendo a este tema @eatskolnikov ya es posible tomar este issue?

**Comment by eatskolnikov** [comment]
@christianrd dale, ya tenemos el servicio de slack creado. Hace falta crear el servicio de github y hacer los otros pasos.

**Comment by christianrd** [comment]
@eatskolnikov estoy trabajando en esto.

**Comment by codercampos** [comment]
@christianrd @eatskolnikov ya no se seguira esta feature?

**Comment by christianrd** [comment]
@codercampos si alguien lo puede tomar que lo tome, esta libre, no podré colaborar hasta finales de octubre.
